app/models/token.rb

```diff
@@ -1,4 +1,4 @@
-class InvalidToken < ActiveRecord::Base
+class Token < ActiveRecord::Base
```

Comment:
**Code Quality - Nitpick**
Rename `invalid_tokens` table to `tokens` for scalability.